File: java/10-messaging-rsocket/users/src/test/java/com/apress/users/UsersSmokeTests.java
package com.apress.users;

import com.apress.users.service.UserService;
import org.junit.jupiter.api.Test;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.boot.test.context.SpringBootTest;

import static org.assertj.core.api.Assertions.assertThat;


@SpringBootTest
class UsersSmokeTests {

    @Autowired
    private UserService userService;

    @Test
    public void contextLoads() throws Exception {
        assertThat(userService).isNotNull();
    }

}

File: kotlin/01-quick-start/users/src/test/kotlin/com/apress/users/UsersSmokeTests.kt
package com.apress.users

import org.assertj.core.api.Assertions
import org.junit.jupiter.api.Test
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.test.context.SpringBootTest

@SpringBootTest
class UsersSmokeTests {

    @Autowired
    private val controller: UsersController? = null

	@Test
	fun contextLoads() {
        Assertions.assertThat(controller).isNotNull
	}

}
